### DISCUSSION TITLE
Heatmap customization

### DISCUSSION THREAD

**Discussion by Fgerthoffert** [reaviz/reaviz#70] [Q&A]
Hi,

I'm new to reaviz and have been struggling a bit to customize the chart.

In the following chart, I'd like to:
<img width="611" alt="Screen Shot 2022-05-08 at 11 44 27 PM" src="https://user-images.githubusercontent.com/5667028/167337182-b96eca71-4288-4f5b-b826-f7bcdc845bfb.png">

- Display the full content of the Y Axis without shrinking, or alternatively to be able to provide a larger width (i.e. 300px wide ) 
- Display the cell value inside the cell (instead of having to hover to see the tooltip)

Thanks

**Comment by amcdnl** [comment]
Can you make a Codesandbox example?

**Reply by Fgerthoffert** [reply]
Here it is: https://codesandbox.io/s/little-thunder-e0bgrj?file=/src/App.js:2195-2235

Would you also know where this error is coming from when hovering over the chart (I'm also getting that error locally)?

```
NotFoundError
Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
```
